pkg/forwarder/forwarder_health.go

```diff
@@ -106,6 +110,24 @@ func (fh *forwarderHealth) healthCheckLoop() {
 	}
 }
 
+//populates a map containing API Endpoints per API keys that belongs to the forwarderHealth struct
+func (fh *forwarderHealth) setKeysPerAPIEndpoints() {
```

Comment:
Could we rename this to something a bit more explicit. Maybe `computeDomainsURL` or similar ?